### PR TITLE
Adding context to salaries pages re maintained schools

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/mI6qifQQ/4773-add-content-around-maintained-schools-on-salaries-pages

### Context

We don't highlight on our salaries pages that the figures apply to maintained schools - schools such as academies can set their own pay scales. We should make this clear to users.

### CODE REVIEW

File: app/views/content/salaries-and-benefits.md [hunk 1]
```diff
@@ -52,11 +52,13 @@ keywords:
 
 For helping to shape the next generation, you’re entitled to a competitive salary, generous holidays, and a substantial pension.
 
-Figures are for the 2022/23 academic year.
+Figures are for the 2022/23 academic year and apply to maintained primary, secondary and special schools in England (schools funded by local authorities).
```
Comment (gemmadallmandfe) [inline app/views/content/salaries-and-benefits.md:55]: @EllieNodder Is it worth bolding `Figures are for the 2022/23 academic year` so it doesn't get lost? But if this is a content no-no, happy as is!

File: app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
         <%= render Content::ChecklistCollageComponent.new(
           checklist: [
-            "You will earn at least £28k (or more in London) if you're just starting as a teacher.",
+            "You'll earn at least £28k (or more in London) if you're just starting as a teacher in a maintained primary, secondary or special school in England (schools funded by local authorities).",
```
Comment (gemmadallmandfe) [inline app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb:12]: @EllieNodder I wonder if this is a bit overkill for the landing page? The advertising team don't include this caveat in their ads

We link through to the salaries and benefits page which has this detail so we are not hiding it

Comment (EllieNodder) [reply app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb:12]: worth leaving the landing page as is then?

File: app/views/content/salaries-and-benefits.md (hunk 1 above)
Comment (EllieNodder) [reply app/views/content/salaries-and-benefits.md:55]: hmm we don't really use bold to emphasise text in regular content pages (and the gov.uk style guide says to use it sparingly) so would be inclined not to do so here

Comment (philspray) [reply app/views/content/salaries-and-benefits.md:55]: Is it worth splitting into 2 sentences? 

```suggestion
Figures are for the 2022/23 academic year. 

These salaries apply at maintained primary, secondary and special schools in England (schools funded by local authorities).
```

**Review by philspray — COMMENTED** [review]
Just one suggestion, otherwise looks good to me.

**Review by philspray — APPROVED** [review]
Looks good to me

**Pull request merged**